src/encoded/static/components/statuslabel.js

```diff
@@ -8,47 +8,79 @@ import * as globals from './globals';
 // property has a name matching the @type of each kind of object. Within that property are objects of
 // arrays; each array defining the statuses available at each logged-in level represented by that
 // object.
+
+const datasetObjectStatuses = {
+    external: [
+        'released',
+        'archived',
+        'revoked',
+    ],
+    consortium: [
+        'proposed',
+        'started',
+        'submitted',
+    ],
+    administrator: [
+        'deleted',
+        'replaced',
+    ],
+};
+
 const objectStatuses = {
-    Experiment: {
+    AntibodyCharacterization: {
         external: [
-            'released',
-            'archived',
-            'revoked',
+            'compliant',
+            'exempt from standards',
+            'not compliant',
+            'not reviewed',
+            'not submitted for review by lab',
         ],
         consortium: [
-            'proposed',
-            'started',
-            'submitted',
+            'in progress',
+            'pending dcc review',
         ],
-        admin: [
+        administrator: [
             'deleted',
-            'replaced',
         ],
     },
+    Experiment: datasetObjectStatuses,
 };
 
 
-// Defines the order of the viewing access of the different logged-in states, and has to match the
-// order of arrays in each @type property in `objectStatuses`.
-const objectStatusLevelOrder = ['external', 'consortium', 'admin'];
+// Defines the order of the viewing access of the different logged-in states.
+const objectStatusLevels = ['external', 'consortium', 'administrator'];
+
+
+/**
+ * Maps the current session information from the <App> React context to an access level from the
+ * objectStatusLevels array.
+ *
+ * @param {object} session - From encoded session context
+ * @param {object} sessionProperties - From encoded session_properties context
+ */
+export const sessionToAccessLevel = (session, sessionProperties) => {
+    const loggedIn = !!(session && session['auth.userid']);
+    const administrativeUser = loggedIn && !!(sessionProperties && sessionProperties.admin);
```

Comment:
Use `Boolean(expr)` instead of `!!expr` for readability. (Turn on https://eslint.org/docs/rules/no-implicit-coercion if we want to enforce this.)